Resources/config/exception_listener.xml

```diff
@@ -5,13 +5,22 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="fos_rest.exception_listener" class="FOS\RestBundle\EventListener\ExceptionListener">
-            <tag name="kernel.event_subscriber" />
+        <service id="security.exception_listener" class="FOS\RestBundle\EventListener\ExceptionListener">
```

Comment:
Ah ok no problem - I'll see if I can get this updated later today. Thanks for the feedback